src/main/java/frc/robot/subsystems/IntakeSubsystem.java

```diff
@@ -1,36 +1,31 @@
 package frc.robot.subsystems;
 
-import com.ctre.phoenix.motorcontrol.ControlMode;
 import com.ctre.phoenix.motorcontrol.InvertType;
 import com.ctre.phoenix.motorcontrol.VictorSPXControlMode;
-import com.ctre.phoenix.motorcontrol.can.WPI_VictorSPX;
-import com.revrobotics.CANSparkMax;
-import com.revrobotics.CANSparkMaxLowLevel.MotorType;
-
+import com.ctre.phoenix.motorcontrol.can.VictorSPX;
 import edu.wpi.first.wpilibj.DigitalInput;
 import edu.wpi.first.wpilibj.DoubleSolenoid;
-import edu.wpi.first.wpilibj2.command.InstantCommand;
 import edu.wpi.first.wpilibj2.command.SubsystemBase;
-import edu.wpi.first.wpilibj2.command.button.Trigger;
-
 import static frc.robot.Constants.*;
 
 public class IntakeSubsystem extends SubsystemBase {
-    private WPI_VictorSPX upperIntakeMotor = new WPI_VictorSPX(UPPER_INTAKE_MOTOR);
-    private CANSparkMax bottomIntakeMotor = new CANSparkMax(BOTTOM_INTAKE_MOTOR, MotorType.kBrushed);
+    private VictorSPX upperIntakeMotor = new VictorSPX(UPPER_INTAKE_MOTOR);
+    private VictorSPX bottomIntakeMotor = new VictorSPX(BOTTOM_INTAKE_MOTOR);
     private DigitalInput intakeSwitch = new DigitalInput(INTAKE_SWITCH);
-    private DoubleSolenoid intakeSolenoid = new DoubleSolenoid(INTAKE_SOLENOID_FWD, INTAKE_SOLENOID_REV);
+    private DoubleSolenoid intakeSolenoid = new DoubleSolenoid(DOUBLE_SOLENOID);
     private Trigger intakeSwitchButton = new Trigger(intakeSwitch::get);
 
     private int ballsCollected = 0;
     private boolean isAlreadyPushed = false;
 
     public IntakeSubsystem() {
-        // upperIntakeMotor.setInverted(InvertType.OpposeMaster);
-        intakeSwitchButton.whenActive(new InstantCommand(() -> { 
-            ballsCollected++;
-            System.out.println("Ball count: " + ballsCollected);
-        }));
+        upperIntakeMotor.follow(bottomIntakeMotor);
+        upperIntakeMotor.setInverted(InvertType.OpposeMaster);
+        intakeSwitchButton.whenActive(new InstantCommand(() -> ballsCollected++));
```

Comment:
This is probably more a design flaw, but the balls collected shouldn't rely on the button. Does `whenActive` happen while held or just when it's pressed once?

Reply:
In theory, if this button is a debounced limit switch type thing, this seems ok to me